osfoffline/sync/ext/watchdog.py

```diff
@@ -62,16 +62,21 @@ def dispatch(self, event):
             logger.debug('Event cache: {}'.format(self._event_cache))
 
             self.timer.cancel()
-            self.timer = threading.Timer(2, self.flush)
+            self.timer = threading.Timer(settings.EVENT_DEBOUNCE, self.flush)
             self.timer.start()
 
+    def _sorted_event_cache(self):
```

Comment:
Given that this doesn't sort the event cache, would you mind renaming this to eg `sorted_create_cache`?